### PR TITLE
feat: pagination of regions applied in start

### PR DESCRIPTION
**Pull request by Xayrulloh**

### CODE REVIEW

File: scenes/start.ts [hunk 1]
```diff
@@ -27,33 +27,68 @@ scene.step(async (ctx) => {
...
 
 // fasting
 scene.wait('fasting').on('callback_query:data', async (ctx) => {
-  if (!ctx.session.regionId.includes(+ctx.update.callback_query.data)) {
-    return ctx.answerCallbackQuery(HLanguage('wrongSelection'))
-  }
+  const inputData = ctx.update.callback_query.data
 
-  await ctx.answerCallbackQuery()
+  if (ctx.session.regionId.includes(+inputData) || ['<', '>'].includes(inputData)) {
```
Comment (toshmukhamedov) [inline scenes/start.ts:40]: How does `regionId` can be an array?

File: scenes/start.ts [hunk 2]
```diff
@@ -27,33 +27,68 @@ scene.step(async (ctx) => {
...
+      ctx.session.message = message
+      ctx.session.buttons = buttons
+
+      await ctx.editMessageText(message, { reply_markup: buttons })
+      ctx.scene.resume()
+
+    }
+  } else {
+
+    await ctx.answerCallbackQuery(HLanguage('wrongSelection'))
```
Comment (toshmukhamedov) [inline scenes/start.ts:88]: first of all, check this case and answer callback query to reduce nested scopes

**Review by toshmukhamedov — CHANGES_REQUESTED** [review]

File: scenes/start.ts (hunk 1 above)
Comment (Xayrulloh) [reply scenes/start.ts:40]: copy/paste issue

File: scenes/start.ts (hunk 2 above)
Comment (Xayrulloh) [reply scenes/start.ts:88]: copy/paste

Comment (toshmukhamedov) [reply scenes/start.ts:88]: doesn't mean you can't fix it

File: scenes/start.ts (hunk 1 above)
Comment (Xayrulloh) [reply scenes/start.ts:40]: wuni anu usuldan qib fix qvorolismi pls. man endi 17 regiondaman teysam yana 0 dan ketadi

File: scenes/start.ts (hunk 2 above)
Comment (Xayrulloh) [reply scenes/start.ts:88]: bunyam if it's possible

File: scenes/start.ts (hunk 1 above)
Comment (toshmukhamedov) [reply scenes/start.ts:40]: what method?)

Comment (Xayrulloh) [reply scenes/start.ts:40]: hold your horses. Man i think its not a bug, (feature)

Comment (Xayrulloh) [reply scenes/start.ts:40]: image
as u can see it's possible

File: scenes/start.ts (hunk 2 above)
Comment (Xayrulloh) [reply scenes/start.ts:88]: > first of all, check this case and answer callback query to reduce nested scopes

yes it is

File: scenes/start.ts (hunk 1 above)
Comment (Xayrulloh) [reply scenes/start.ts:40]: I recommend you to rereview

Comment (toshmukhamedov) [reply scenes/start.ts:40]: why?

File: scenes/start.ts (hunk 2 above)
Comment (Xayrulloh) [reply scenes/start.ts]: @toshmukhamedov

**Review by toshmukhamedov — APPROVED** [review]

**Pull request merged**